packs/equipment/dinosaur-boots-greater.json

```diff
@@ -9,7 +9,7 @@
         },
         "containerId": null,
         "description": {
-            "value": "<p>The tough, scaled leather of these heavy boots comes from a mighty dinosaur, granting you the steadiness of a lumbering beast. Any time an action or effect would cause you to make a forced movement, roll a @Check[flat|dc:17] check. On a success, the forced movement fails to affect you.</p>\n<hr />\n<p><strong>Activate</strong> <span class=\"action-glyph\">1</span> command (magical, morph, transmutation)</p>\n<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p><strong>Effect</strong> You grow to size Huge if you choose, with the effects of 4th-rank @UUID[Compendium.pf2e.spells-srd.Item.Enlarge] for 1 minute. During that time, you gain dinosaur features; your legs and feet transform into a dinosaur's. You gain a foot unarmed attack that has the same statistics as your fist unarmed attack, except its damage die is increased from 1d4 to 1d8. During the duration, you can use the @UUID[Compendium.pf2e.bestiary-ability-glossary-srd.Item.Trample] 3-action activity to Trample creatures one size smaller than you or smaller. This deals an amount of damage to each creature equal to that of your foot unarmed attack (including any extra weapon damage dice, bonuses, or additional damage as normal), with a @Check[reflex|dc:27|basic|name:Dinosaur Trample|traits:uncommon,invested,magical,transmutation|showDC:all] save. You can Dismiss the activation.</p>"
+            "value": "<p>The tough, scaled leather of these heavy boots comes from a mighty dinosaur, granting you the steadiness of a lumbering beast. Any time an action or effect would cause you to make a forced movement, roll a @Check[flat|dc:17] check. On a success, the forced movement fails to affect you.</p>\n<hr />\n<p><strong>Activate</strong> <span class=\"action-glyph\">1</span> command (magical, morph, transmutation)</p>\n<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p><strong>Effect</strong> You grow to size Huge if you choose, with the effects of 4th-rank @UUID[Compendium.pf2e.spells-srd.Item.Enlarge] for 1 minute. During that time, you gain dinosaur features; your legs and feet transform into a dinosaur's. You gain a foot unarmed attack that has the same statistics as your fist unarmed attack, except its damage die is increased from 1d4 to 1d8. During the duration, you can use the @UUID[Compendium.pf2e.bestiary-ability-glossary-srd.Item.Trample] 3-action activity to Trample creatures one size smaller than you or smaller. This deals an amount of damage to each creature equal to that of your foot unarmed attack (including any extra weapon damage dice, bonuses, or additional damage as normal), with a @Check[reflex|dc:27|basic|name:Dinosaur Trample|traits:invested,magical,transmutation|showDC:all] save. You can Dismiss the activation.</p>"
```

Comment:
```suggestion
            "value": "<p>The tough, scaled leather of these heavy boots comes from a mighty dinosaur, granting you the steadiness of a lumbering beast. Any time an action or effect would cause you to make a forced movement, roll a @Check[flat|dc:17] check. On a success, the forced movement fails to affect you.</p>\n<hr />\n<p><strong>Activate</strong> <span class=\"action-glyph\">1</span> command (magical, morph, transmutation)</p>\n<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p><strong>Effect</strong> You grow to size Huge if you choose, with the effects of 4th-rank @UUID[Compendium.pf2e.spells-srd.Item.Enlarge] for 1 minute. During that time, you gain dinosaur features; your legs and feet transform into a dinosaur's. You gain a foot unarmed attack that has the same statistics as your fist unarmed attack, except its damage die is increased from 1d4 to 1d8. During the duration, you can use the @UUID[Compendium.pf2e.bestiary-ability-glossary-srd.Item.Trample] 3-action activity to Trample creatures one size smaller than you or smaller. This deals an amount of damage to each creature equal to that of your foot unarmed attack (including any extra weapon damage dice, bonuses, or additional damage as normal), with a @Check[reflex|dc:27|basic|name:Dinosaur Trample|traits:magical,transmutation|showDC:all] save. You can Dismiss the activation.</p>"
```